client/static_src/components/tracker.vue

```diff
@@ -88,24 +135,64 @@ export default {
   },
   data() {
     return {
-      task: null,
-      project: null,
-      note: null,
+      task: getFromLocalStorage("tracker_task", null),
```

Comment:
read tracker state from local storage. i sometimes stopped and lost my timer by accident